openassessment/xblock/ui_mixins/mfe/page_context_serializer.py

```diff
@@ -260,4 +276,4 @@ def get_submission(self, instance):
 
             return AssessmentResponseSerializer(instance.api_data, context=self.context).data
```

Comment:
The reason we currently have separate `assessment` and `submission` serializers is because the data we are returning as `submission` comes from different places depending on if you are getting your response or someone else's response to assess.

What is returned under `assessments` should be a different shape, scores only as described [here](https://2u-internal.atlassian.net/wiki/spaces/AU/pages/499613875/ORA+MFE+-+Data+Contracts#Received-Assessments).